packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/simulation/Simulation.java

```diff
@@ -85,7 +85,6 @@ public class Simulation extends TerariumAsset {
 
 	@OneToMany(mappedBy = "simulation", fetch = FetchType.EAGER, cascade = CascadeType.ALL)
 	@OrderBy("createdOn DESC")
-	@ToString.Exclude
```

Comment:
Is the infinite loop on `toString` no longer a problem here?

Reply:
Doesn't seem to be if we use `JsonManagedReference` and `JsonBackReference`